spk/syncthing/src/service-setup.sh

```diff
@@ -13,6 +13,13 @@ service_prestart ()
         SERVICE_COMMAND="${SERVICE_COMMAND} ${SYNCTHING_OPTIONS}"
     fi
 
+    # If the system has a TLS certificate for us, force its usage
+    cert_dir=/usr/local/etc/certificate/${SYNOPKG_PKGNAME}/${SERVICE_CERT}
+    if [ -f ${cert_dir}/cert.pem -a -f ${cert_dir}/privkey.pem ]; then
+    	ln -sf ${cert_dir}/cert.pem ${SYNOPKG_PKGVAR}/https-cert.pem
```

Comment:
Same indentation request here - keep them consistent in the whole script

Reply:
Yea, sorry.  My editor is configured to use one tab instead of eight spaces when possible.  Fixed now.